### PR TITLE
10234 fix LS-PrePost

### PR DESCRIPTION
**Pull request by tsubic**
Fixed older versions of easyconfig files for LS-PrePost and added a new version (4.7).

### CODE REVIEW

File: easybuild/easyconfigs/l/LS-PrePost/LS-PrePost-4.7.eb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+easyblock = "Tarball"
+
+name = 'LS-PrePost'
+version = '4.7'
```
Comment (Micket) [inline easybuild/easyconfigs/l/LS-PrePost/LS-PrePost-4.7.eb]: Looks like they have started to use all 3 numbers for versions, which would solve problem with different date suffixes for the same "version". I think we should use 4.7.8 going forward.

File: easybuild/easyconfigs/l/LS-PrePost/LS-PrePost-4.7.eb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+homepage = 'http://lstc.com/products/ls-prepost'
+description = """LS-PrePost is an advanced pre and post-processor that is delivered free with LS-DYNA."""
+
+toolchain = SYSTEM
+
+source_urls = ['http://ftp.lstc.com/anonymous/outgoing/lsprepost/%(version)s/linux64/']
```
Comment (Micket) [inline easybuild/easyconfigs/l/LS-PrePost/LS-PrePost-4.7.eb]: and we should switch to %(version_major_minor)s here

File: easybuild/easyconfigs/l/LS-PrePost/LS-PrePost-4.7.eb (hunk 1 above)
Comment (tsubic) [reply easybuild/easyconfigs/l/LS-PrePost/LS-PrePost-4.7.eb]: ok, will fix it, let's hope that they wont change dates anymore

File: easybuild/easyconfigs/l/LS-PrePost/LS-PrePost-4.7.eb (hunk 2 above)
Comment (tsubic) [reply easybuild/easyconfigs/l/LS-PrePost/LS-PrePost-4.7.eb]: so, having one variable "version" which woudl in this case be 4.7.8, and one variable "version_major_minor" which would be just 4.7?

The problem is that in the url they use just 4.7

Comment (tsubic) [reply easybuild/easyconfigs/l/LS-PrePost/LS-PrePost-4.7.eb]: thanks for the comment, fixed

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
hebbe-c1 - Linux centos linux 7.7.1908, Intel Core Processor (Haswell, no TSX), Python 2.7.5
See https://gist.github.com/7b87d67629d400320a98184921bfd974 for a full test report.

**Comment by Micket** [comment]
Going in, thanks @tsubic!

**Pull request merged**